client/actions/events.js

```diff
@@ -56,6 +56,13 @@ export function uploadFilesAndSaveEvent(newEvent) {
     newEvent = cloneDeep(newEvent) || {}
     return (dispatch) => (
         Promise.resolve((() => {
+            // Validate event links
+            if (newEvent.links && newEvent.links.length > 0) {
+                newEvent.links = newEvent.links.filter(
+                    (l) => (l && l !== 'Paste link')
```

Comment:
Why not using a `placeholder` attribute for this purpose? then you wouldn't have to make this check.
Also this would risky to use an hard coded string here. If the text is changed in the other file, this instruction won't be relevant anymore.